app/src/main/java/sun/trainingcourse/musicapp/service/MusicControlService.kt

```diff
@@ -0,0 +1,101 @@
+package sun.trainingcourse.musicapp.service
+
+import android.app.Service
+import android.content.Intent
+import android.media.MediaPlayer
+import android.os.Binder
+import android.os.IBinder
+import sun.trainingcourse.musicapp.data.local.MusicData
+import sun.trainingcourse.musicapp.data.model.Music
+
+private const val EXTRA_MUSIC_POSITION = "position"
+private const val EXTRA_MUSIC = "music"
+
+class MusicControlService : Service() {
+
+    private var mMediaPlayer: MediaPlayer? = null
+    private var mMusic: Music? = null
+    private var mPosition: Int = 0
+    private var mServiceCallBack: MusicServiceCallBack? = null
+    private lateinit var mListMusic: ArrayList<Music>
```

Comment:
Mình suggest khái báo như này nhé 
```suggestion
    private var mediaPlayer: MediaPlayer? = null
    private var music : Music? = null
    private var position: Int = 0
    private var someThingListener: MusicServiceCallBack? = null
    private val musics = mutableListOf<Music>()
```